checkbox-ng/checkbox_ng/launcher/checkbox_cli.py

```diff
@@ -141,12 +141,7 @@ def main():
     subcmd = commands[args.subcommand]()
     subcmd.register_arguments(subcmd_parser)
     sub_args = subcmd_parser.parse_args(sys.argv[subcmd_index + 1 :])
-    sa = SessionAssistant(
-        "com.canonical:checkbox-cli",
-        "0.99",
-        "0.99",
-        ["restartable"],
-    )
+    sa = SessionAssistant()
```

Comment:
I had been copy-pasting the previous blob of text every time I wanted to test something without realizing the parameters were not useful in most of the cases :S

Reply:
Pierre told me on MM:

>oh sorry, it was just a comment for myself, mostly
>nothing to change